### PR TITLE
Feature 123 moment to dayjs

### PR DESCRIPTION
**Pull request by Martijn0405**
### What is the change?

All usecases of momentJS are replaced with dayJS, except the momentLocalizer specifically built for react-big-calendar. There was no clear replacement for moment at this point in the code. Replacing the momentLocalizer could be a future issue to fix.

### CODE REVIEW

**Comment by harshith-venkatesh** [comment]
@Martijn0405 , I don't see any issue created to address this.
Could you please share some documents or articles that is recommending this, or that could be backed in this case

**Comment by Martijn0405** [comment]
@harshith-venkatesh I managed to find a solution that does not give development issues and runs the project. I find the documentation of localizers in react-big-calendar quite vague to guarantee there won't be errors in this implementation. Could you please review the code?

**Comment by iRohitGaur** [comment]
> @harshith-venkatesh I managed to find a solution that does not give development issues and runs the project. I find the documentation of localizers in react-big-calendar quite vague to guarantee there won't be errors in this implementation. Could you please review the code?

Hello @Martijn0405 , can you please take the latest pull and resolve the conflicts?

File: app/components/common/rdsCalendar/index.tsx [hunk 1]
```diff
@@ -1,17 +1,79 @@
...
+      return data ? data.firstDayOfWeek() : 0;
+    },
+
+    format(value, format, culture) {
+      return locale(dayjs(value), culture).format(format);
+    },
+  });
+};
+
+const localizer = dayjsLocalizer();
```
[the remark this answers is not in the document]
Comment (khushi818) [reply app/components/common/rdsCalendar/index.tsx:76]: yes i am doing that

File: tailwind.config.js [hunk 2]
```diff
@@ -4,12 +4,33 @@ module.exports = {
...
+          med: '#78716C',
+          dark: '#44403C',
+        },
         resolutionBlue: '#041484',
         razzmatazz: '#E30464',
         feijoa: '#96D882',
         fern: '#6DB470',
       },
+      fontSize: {
+        14: '3.5rem',
```
Comment (isVivek99) [inline tailwind.config.js:22]: i don't think we need font sizes, can we remove these?

File: tailwind.config.js [hunk 3]
```diff
@@ -4,12 +4,33 @@ module.exports = {
...
   theme: {
     extend: {
+      borderRadius: {
+        16: '4rem',
+      },
       colors: {
+        grey: {
+          DEFAULT: '#D6D3D1',
+          med: '#78716C',
+          dark: '#44403C',
```
Comment (isVivek99) [inline tailwind.config.js:14]: also this color `grey`

File: tailwind.config.js [hunk 4]
```diff
@@ -4,12 +4,33 @@ module.exports = {
...
+      },
+      spacing: {
+        1: '0.25rem',
+        2: '0.5rem',
+        4: '1rem',
+        8: '2rem',
+        16: '4rem',
+        24: '6rem',
+        32: '8rem',
+      },
```
Comment (isVivek99) [inline tailwind.config.js:33]: not required

File: .prettierrc.json [hunk 5]
```diff
@@ -1,4 +1,7 @@
 {
+  "semi": true,
+  "trailingComma": "all",
+  "singleQuote": true,
   "printWidth": 100,
-  "singleQuote": true
```
Comment (isVivek99) [inline .prettierrc.json:3]: why are we removing this?

Comment (isVivek99) [reply .prettierrc.json:3]: i think you need to take the latest pull from the `develop` branch

**Comment by harshith-venkatesh** [comment]
Hey @Martijn0405 , 
@khushi818  PR is an extension of this PR
https://github.com/Real-Dev-Squad/website-calendar/pull/154
Keeping you in loop

**Comment by harshith-venkatesh** [comment]
Thanks for helping us here @Martijn0405 , We would be closing this PR as it is being addressed

**Pull request closed without merging**